Sources/Rx+Combine/Infallible+Combine.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Infallible+Combine.swift
+//  RxCombine
+
+#if canImport(Combine)
+import Combine
+import RxSwift
+
+@available(OSX 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
+public extension Infallible {
+    /// An `AnyPublisher` of the underlying Observable's Element type
+    /// so the Infallible pushes events to the Publisher.
+    var publisher: AnyPublisher<Element, Swift.Never> {
+        RxInfalliblePublisher(upstream: self).eraseToAnyPublisher()
+    }
```

Comment:
this is great. Another option could probably just do `asPublisher().assertNoFailure("Infallible should not fail")` and that way you don't need another custom Publisher type. WDYT?

Reply:
Yeah. Could do that too. But RxInfallibleSubscriber already exists so why not use that? Seems a cleaner solution.
Either way. If you prefer assertNoFailure I’ll update to that

Reply:
No, I was referring to RxInfallibleSubscriber which does already exist but is not used anywhere.

Reply:
Anyway, I've changed it to use assertNoFailure as you suggested